_layouts/py1-lesson.html

```diff
@@ -0,0 +1,76 @@
+---
+layout: default
+fw: true
+---
+<div class="py1-lesson-page-wrapper">
+    <div class="py1-lesson-sidebar">
+        <div class="py1-section1">
+            <h1 class="py1-lesson-header"> {{ page.num }}. {{ page.title }} </h1>
+            <p class="py1-lesson-page-description">
+                {{ content }}
+            </p>
+
+            <p class="py1-lesson-page-agenda">
+                <b> agenda </b>
+                {{ page.agenda }}
+            </p>
+
+            <div class="py1-lesson-page-action-bar">
+                {% if page.slides_link %}
+                <a href="{{ page.slides_link }}" target="_blank">slides</a>
+                {% endif %}
+
+                {% if page.worksheet_link %}
+                <a href="{{ page.worksheet_link }}" target="_blank">worksheet</a>
+                {% endif %}
+
+                <span class="py1-lesson-duration">{{ page.length }} min</span>
+            </div>
+        </div>
+
+        {% if page.next_lesson %}
+        <div class="py1-lesson-page-next">
+            <a href="{{site.baseurl}}/classes/py1/{{page.next_link}}">next lesson: {{ page.next_lesson }}</a>
+        </div>
+        {% endif %}
```

Comment:
This next lesson button is really far down on the screen (if the lesson content is really large); could we have it just immediately follow the rest of the content vertically?